scripts/travisci/check_precommit.sh

```diff
@@ -1,15 +1,8 @@
 #!/usr/bin/env bash
 if [[ "${TRAVIS_PULL_REQUEST}" != "false" && "${TRAVIS}" == "true" ]]; then
-  files_changed=$(git diff "${TRAVIS_COMMIT_RANGE}" --name-only \
-    | grep ".*\.py")
-  if [[ ! -z "${files_changed}" ]]; then
-    pre-commit run --files  ${files_changed}
-  fi
+  pre-commit run --source ${TRAVIS_COMMIT_RANGE%...*} --origin ${TRAVIS_COMMIT_RANGE#*...}
```

Comment:
Did you send a few intentionally broken commits to make sure it's still actually catching errors?

You can ignore precommit hooks using `git commit --no-verify`

Reply:
that's helpful, thanks!